scripts/extensions/markForUser/src/managed-marked-user.tsx

```diff
@@ -1,7 +1,7 @@
 import {ISuperdesk, IArticle} from 'superdesk-api';
 import {getMarkForUserModal} from './get-mark-for-user-modal';
 
-export function manageMarkedUserForSingleArticle(superdesk: ISuperdesk, article: IArticle) {
+export function manageMarkedUserForSingleArticle(superdesk: ISuperdesk, article: IArticle, ignoreLock = false) {
```

Comment:
Are you sure you need `ignoreLock`? It was already possible to change marked user from topbar in article edit view even when an article is opened for editing.

Reply:
How? I can't do that

And without the `ignoreLock` I get:

![Captura de pantalla 2019-08-07 a las 14 40 59](https://user-images.githubusercontent.com/4324982/62623611-910b3180-b921-11e9-9aa7-99254ffca81c.png)

Reply:
Never mind. I found a bug where no underscore prefixed properties weren't present on article object. That's why it worked for me without introducing `ignoreLock`. Keep it :)